lib/slack/web/api/patches/chat.1.patch

```diff
@@ -50,7 +50,7 @@ index d090ae2..50186d5 100644
            # @see https://github.com/slack-ruby/slack-api-ref/blob/master/methods/chat/chat.update.json
            def chat_update(options = {})
              throw ArgumentError.new('Required arguments :channel missing') if options[:channel].nil?
-+            throw ArgumentError.new('Required arguments :text, :attachments or :blocks missing') if options[:text].nil? && options[:attachments].nil? && options[:blocks].nil?
++            throw ArgumentError.new('Required arguments :text, :attachments, :blocks or :reply_broadcast missing') if options[:text].nil? && options[:attachments].nil? && options[:blocks].nil? && options[:reply_broadcast].nil?
              throw ArgumentError.new('Required arguments :ts missing') if options[:ts].nil?
              options = options.merge(channel: conversations_id(options)['channel']['id']) if options[:channel]
 +            # attachments must be passed as an encoded JSON string
```

Comment:
This looks suspicious in the patch file. Does this patch apply clean? Verify with `rake api:update`.

Reply:
I didn't add this line 🤔 
`rake api:update` worked with no diffs on b964bb06fd1da10b58bdea9e4e58c19b6c888bf9 .